src/openloco/windows/LandscapeGeneration.cpp

```diff
@@ -186,9 +187,9 @@ namespace openloco::ui::windows::LandscapeGeneration
         {
             common::prepare_draw(window);
 
-            commonFormatArgs[0] = *scenarioStartYear;
+            commonFormatArgs[0] = s5::getOptions().scenarioStartYear;
```

Comment:
I think it would make the code clearer if there were one `options` variable. One reference in each method or even namespace global, that is.